docs/external-node.md

```diff
@@ -266,13 +266,24 @@ please refer to the [getting-started guide](getting-started.md#open-vswitch).
    ```
 
 2. The `antrea-agent.conf` file specifies agent configuration parameters. Copy
```

Comment:
I feel we should revise the paragraph, like:

2. Copy configuration files to the VM, including [`antrea-agent.conf`](../build/yamls/externalnode/conf/antrea-agent.conf), which specifies agent configuration parameters; `antrea-agent.antrea.kubeconfig` and `antrea-agent.kubeconfig`, which were generated in steps 4 and 5 of [Prerequisites on Kubernetes cluster](#prerequisites-on-kubernetes-cluster).

The starting sentence should talk about an operation, as other steps.

Reply:
Updated it based on your suggestion.